src/components/Label/Label.tsx

```diff
@@ -1,45 +1,46 @@
 import * as React from "react";
+
 import bem from "../../utils/bem";
 
-export interface LabelOptions {
-  labelContent?: JSX.Element;
-  id?: string;
-  htmlFor?: string;
-}
+type optReqFlagType = "Required" | "Optional" | "" | undefined;
 
 interface LabelProps {
   /** Additional attributes to pass to the <label> tag */
   attributes?: { [key: string]: any };
-
   /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
   blockName?: string;
   /** className that appears in addition to "label" */
   className?: string;
-  /** The id of the html element that this Label is describing */
+  /** The id of the html element that this `Label` is describing */
   htmlFor?: string;
   /** ID that other components can cross reference for accessibility purposes */
   id?: string;
   /** Modifiers for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
   modifiers?: string[];
-  /** Displays "Required" or "Optional" string alongside label */
-  optReqFlag?: string;
+  /** Displays "Required" or "Optional" string alongside the label */
+  optReqFlag?: optReqFlagType;
 }
 
-/** A Label for form inputs.
- * It should never be used alone.
+/**
+ * A label for form inputs. It should never be used alone.
  */
 export default function Label(props: React.PropsWithChildren<LabelProps>) {
   const {
     attributes,
     blockName,
+    children,
     className,
     htmlFor,
     id,
-    modifiers = [],
     optReqFlag,
+    modifiers = [],
```

Comment:
Why was `modifiers` moved to the end of the list of props?

Reply:
Oh, not sure. I'll move it back.